src/js/components/Anchor/index.d.ts

```diff
@@ -32,7 +32,9 @@ export interface AnchorProps {
   weight?: 'normal' | 'bold' | number;
 }
 
-declare const Anchor: React.FC<AnchorProps &
-  Omit<JSX.IntrinsicElements['a'], 'color'>>;
+export type AnchorExtendedProps = AnchorProps &
```

Comment:
I think we should export an 'interface' vs a 'type', right @abnersajr ?

Reply:
Yeah, I think it's better to export interfaces instead of types, so it can be consistent with the already merged changes. I've made the changes, it should be correct now!